awx/ui/src/screens/Dashboard/Dashboard.js

```diff
@@ -79,8 +77,8 @@ function Dashboard() {
         <Banner variant="info">
           <Trans>
             <p>
-              <InfoCircleIcon /> A tech preview of the new {brandName} user
-              interface can be found <a href="/ui_next">here</a>.
+              <InfoCircleIcon /> The new user interface can be found{' '}
```

Comment:
Please don't use "here"
Maybe you the link should be "enable new user interface"

Reply:
reworded it, see photo in opening body